README.md

```diff
@@ -204,20 +204,43 @@ Number, maximum amount of concurrent requests. Defaults to `Infinity`.
 
 Plugins allow to extend scraper behaviour
 
-* [Built-in plugins](#built-in-plugins)
-* [Existing plugins](#existing-plugins)
-* [Create plugin](#create-plugin)
-* Create action
-    * [beforeStart](#beforestart)
-    * [afterFinish](#afterfinish)
-    * [error](#error)
-    * [beforeRequest](#beforerequest)
-    * [afterResponse](#afterresponse)
-    * [onResourceSaved](#onresourcesaved)
-    * [onResourceError](#onresourceerror)
-    * [saveResource](#saveresource)
-    * [generateFilename](#generatefilename)
-    * [getReference](#getreference)
+- [website-scraper](#website-scraper)
```

Comment:
There are too many changes here, could you please leave only plugins-related items?